Dockerfile

```diff
@@ -11,6 +11,8 @@ WORKDIR /code
 
 # Install Python dependencies
 RUN pip install pipenv
+
```

Comment:
Whitespace diff

Reply:
There's still a whitespace diff here, not a huge deal but just something we can catch in `git status` / `git diff head` checks.